integration_tests/integration/allocation-complete.spec.ts

```diff
@@ -32,60 +35,67 @@ context('Allocate Complete', () => {
       .should('contain', 'Dylan Adam Armstrong (J678910) has been allocated to John Doe (PO)')
   })
 
-  // it('What happens next with multiple emails supplied, opting out of copy content visible on page', () => {
```

Comment:
If this test is broken and you've completely replaced it, I'd say just delete it instead of leaving it commented

Reply:
It is deleted.